core/capabilities/ccip/ccipevm/executecodec_test.go

```diff
@@ -217,7 +218,7 @@ func TestExecutePluginCodecV1(t *testing.T) {
 }
 
 func Test_DecodeReport(t *testing.T) {
-	ExtraDataCodec := ccipcommon.NewExtraDataCodec(ccipcommon.NewExtraDataCodecParams(ExtraDataDecoder{}, ccipsolana.ExtraDataDecoder{}))
+	ExtraDataCodec := ccipcommon.NewExtraDataCodec(ccipcommon.NewExtraDataCodecParams(ExtraDataDecoder{}, ccipsolana.ExtraDataDecoder{}, ccipaptos.ExtraDataDecoder{}))
```

Comment:
Looks like this list of codec's exists in a few places, probably worth moving into `NewExtraDataCodec` so that it can be updated once.
cc: @prashantkumar1982 @huangzhen1997

Reply:
yes get rid of `ExtraDataCodecParams` is part of our planned work that will be added to https://github.com/smartcontractkit/chainlink/pull/16363.